src/tools/interop/idt/README.md

```diff
@@ -0,0 +1,564 @@
+# Interoperability Debugging Tool
+
+## Overview
+
+The “Interoperability Debugging Tool” (IDT) is a python-based tool that supports
+a variety of commands that are useful in the context of interoperability testing
+of Matter devices and app controllers.
+
+### Discovery
+
+While in discovery mode, the tool displays all Matter devices that are in
+commission and/or operational mode. This is useful to have a clear understanding
+of all Matter devices currently “active” in the testing environment.
+
+See section “4.3. Discovery” of the Matter spec for official documentation.
+
+When run interactively, discovery functions in one of two modes: BLE and DNS-SD.
+
+### Capture
+
+While in capture mode, the tool starts capturing all data of interest (e.g.
+video recording of interactions with the mobile app, logs from all components
+involved, network packets capture, etc.) while a test is being conducted
+manually. It also provides feedback to the user on test setup and execution.
+
+When the test completes, capture mode is stopped and all captured data is zipped
+in a file that can then be sent to all parties involved in investigating any
+issue uncovered via the manual test. Each ecosystem may implement an analysis
+that analyzes capture data, displays info to the user, probes the local
+environment and generates additional artifacts.
+
+## Getting started
+
+## Raspberry Pi installation
+
+### Environment overview
+
+The execution environment of IDT when using Raspberry Pi is shown in the figure
+below.
+
+[TODO] add figure.
```

Comment:
This would be helpful. I'm currently trying to set this up on two Raspberry Pis... One as the "admin" and the other as the "monitor". It would help having a diagram with labels, and then consistently use the terminology in the doc. I was getting a little confused how "Kali", "Raspberry Pi", "Admin machine", "Docker instance", etc. all interact with each other.

Reply:
We'll eventually add this in!